locations/spiders/ikea.py

```diff
@@ -62,16 +62,16 @@ class IkeaSpider(scrapy.Spider):
     def parse(self, response):
         data = response.json()
         for store in data:
-            if "storePageUrl" not in store:
-                continue
             opening_hours = OpeningHours()
-            for day in store["hours"]["normal"]:
+            for day in store["hours"]["normal"] if "hours" in store else []:
                 if day["open"] != "":
                     opening_hours.add_range(
                         day["day"][0:1].upper() + day["day"][1:2].lower(),
```

Comment:
This line can be changed to `day["day"].title(),`

Reply:
My bad, it's actually `day["day"].title()[:2],` that's the whole line.

`title` gives us the first letter in uppercase, than the next letters in lower ("MON" -> "Mon"). And then we spit it to the just to 2 chars, so "Mon"->"Mo".